lib/normalize.js

```diff
@@ -34,6 +34,8 @@ module.exports = function normalize(nemo, _locator) {
   var normalizedLocator;
   if (_locator.constructor === String) {
     locator = _splitLocator(nemo, _locator);
+  } else if (locator instanceof nemo.wd.By.constructor) {
+      return locator; //no need to normalize, WebDriver understands this format.
   }
```

Comment:
This is required because underscore methods like `_find` call `normalize` internally. And when we use something like `nemo.view._find(nemo.view.simple.waitButtonBy())`, we are already passing a normalized locator and there is no need to normalize again. In fact normalizing breaks tests. This change fixes it